helpers/helper-only-numbers/src/index.ts

```diff
@@ -1,3 +1,3 @@
 export default function onlyNumbers(str: string | number) {
-  return String(str).replace(/[^\d]/g, '');
+  return str.toString().replace(/[^\d]/g, '');
```

Comment:
I think that `String(str)` is better because if you receive a number you can't do `.toString()`

Reply:
@hyanmandian `Number` prototype has a `toString` method btw. I also wrote a test that handle this scenario.

https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/Number/toString

Reply:
`String(str)` will behave better if `null` passed as a parameter, but we are using types. I do not know if this is a problem.